styles/globals.css

```
@@ -344,7 +363,14 @@ select:focus {
 }
 
 .prose a {
-  @apply transition-all;
+  @apply text-accent-soft underline transition-all;
+  text-underline-offset: 2px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)